src/pages/403.tsx

```diff
@@ -0,0 +1,12 @@
+import React from 'react';
+
+function Custom403() {
```

Comment:
Perhaps it would be better to follow convention from `Unauthorized` and call this `Forbidden`.

Reply:
Right, I wasn't sure about that, thanks!